src/js/models/tools/ellipse.js

```diff
@@ -47,7 +47,7 @@ define(function (require, exports, module) {
             backspaceKeyPolicy = new KeyboardEventPolicy(UI.policyAction.PROPAGATE_TO_BROWSER,
                 OS.eventKind.KEY_DOWN, null, OS.eventKeyCode.BACKSPACE);
         
-        Tool.call(this, "ellipse", "Ellipse", "ellipseTool", "tool.ellipse.select");
+        Tool.call(this, "ellipse", "Ellipse Tool", "ellipseTool", "tool.ellipse.select");
```

Comment:
If this string (and others elsewhere) are important enough for you to want to add " Tool" here... then they are probably important enough to localize.